authors/apps/authentication/views.py

```diff
@@ -18,16 +28,60 @@ class RegistrationAPIView(APIView):
 
     def post(self, request):
         user = request.data.get('user', {})
-
         # The create serializer, validate serializer, save serializer pattern
         # below is common and you will see it a lot throughout this course and
         # your own work later on. Get familiar with it.
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
-        serializer.save()
-        
+        user = serializer.save()
+        user.is_active = False
+        user.save()
+        current_site = get_current_site(request)
+        response_data = self.send_email_notification(current_site, user)
+        response_data.update(serializer.data)
+        return Response(response_data, status=status.HTTP_201_CREATED)
+
+    def send_email_notification(self, current_site, user):
+        domain = f'http://{current_site.domain}'
+        token = default_token_generator.make_token(user)
+        uid = urlsafe_base64_encode(force_bytes(user.username)).decode('utf-8')
+        print(domain)
```

Comment:
please remove the print statements.

Reply:
thanks for the feedback let me work onthat